### PR TITLE
rec: Mention += in the docs plus an example

### PR DESCRIPTION
**Pull request by omoerbeek**
### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: pdns/recursordist/docs/index.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Introduction
 The PowerDNS Recursor is a high-performance DNS recursor with built-in scripting capabilities.
 It is known to power the resolving needs of over 150 million internet connections.
 
-The documentation is only for the 4.1 series, users of older versions are urged to upgrade!
+The documentation is only for the 4.1 and higher series, users of older versions are urged to upgrade!
```
Comment (Habbie) [inline pdns/recursordist/docs/index.rst:11]: nobody should upgrade to 4.1. Perhaps link to EOL here?

File: pdns/recursordist/docs/settings.rst [hunk 2]
```diff
@@ -12,6 +12,14 @@ As an example:
  - ``serve-rfc1918=off`` or ``serve-rfc1918=no`` means: do not serve those zones.
  - Anything else means: do serve those zones.
 
+You can use ``+=`` syntax to set some variables incrementally, but this
+requires you to have at least one non-incremental setting for the
+variable to act as base setting. This is mostly useful for
+:ref:`setting-include-dir` directive. An example::
+
+  forward-zones = foo.example.com=192.168.100.1;
+  forward-zones += bar.example.com=[1234::abcde]:5353;
```
Comment (Habbie) [inline pdns/recursordist/docs/settings.rst:21]: I don't think this needs the trailing `;` - does that even work?

**Review by Habbie — COMMENTED** [review]
two nits

File: pdns/recursordist/docs/settings.rst (hunk 2 above)
Comment (omoerbeek) [reply pdns/recursordist/docs/settings.rst:21]: Yes it works and it is even handy even you are going to add more line in the future

**Review by Habbie — APPROVED** [review]

**Review by Habbie — APPROVED** [review]

**Pull request merged**